lib/ASN1/Object.php

```diff
@@ -295,6 +299,9 @@ protected static function parseBinaryIdentifier($binaryData, &$offsetIndex)
         }
 
         while (true) {
+            if (strlen($binaryData) <= $offsetIndex) {
+                throw new ParserException('Can not parse binary identifier from data: Offset index larger than input size', $offsetIndex);
```

Comment:
I suggest we add the `(long form)` hint to the exception message as you already did in `parseContentLength`